config/config.yaml

```diff
@@ -124,7 +125,10 @@ params:
   # also, separate capitalised letter flags are required for adapters in
   # the reverse reads of paired end sequencing:
   # * https://cutadapt.readthedocs.io/en/stable/guide.html#trimming-paired-end-reads
-  cutadapt-se: " "
+  
+  cutadapt-se:
+    adapters: "-a AGATCGGAAGAGCACACGTCTGAACTCCAGTCAC"
+    extra: "-q 20"
```

Comment:
Ensure that all changes in the config structure are reflected in workflow/schemas/config.schema.yaml.

Reply:
Noted